src/Messages/Messages.module.css

```diff
@@ -0,0 +1,191 @@
+.messagesPageContainer {
+  max-width: 935px;
+  margin: 80px auto;
+  display: flex;
+}
+
+.userSection {
+  display: flex;
+  width: 350px;
+  border-top: 1px solid #dbdbdb;
+  border-left: 1px solid #dbdbdb;
+  border-right: 1px solid #dbdbdb;
+  padding: 0;
+  margin: 0;
+  height: 60px;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+}
+.sendMessageIcon {
+  padding-right: 20px;
+}
+.usernameIconSection {
+  display: flex;
+  margin: 30px 20px 30px 120px;
```

Comment:
I appreciate the creativity, but this not good. Writing the coordinates manually makes your CSS brittle (it breaks easily). I would add an empty `div` in `userSection`. The give `flex: 1` to everything inside `userSection`. This way you'll have 3 equal parts with the username in the center. 

```css
userSection > * {
	flex: 1
}
```